.github/workflows/tests.yml

```diff
@@ -30,17 +30,18 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/vendor/bundle
-          key: ${{ runner.os }}-${{ matrix.ruby }}-gems-${{ hashFiles('apps/*/Gemfile.lock', 'Gemfile.lock') }}
+          key: ${{ runner.os }}-${{ matrix.ruby }}-unit-gems-${{ hashFiles('apps/*/Gemfile.lock', 'Gemfile.lock') }}
 
       - name: Setup Bundler
         run: |
           bundle config path ~/vendor/bundle
+          bundle install
```

Comment:
Actually I think here is where the CI issues stem from. The CI won't upload a new cache, so I think during this PR - the first cache it uploaded was wrong and now it won't upload a good cache. So I've given it a different name, cache key, so it'll upload a cache with all the gems we want.